lib/plenario2/actions/meta_actions.ex

```diff
@@ -62,8 +67,75 @@ defmodule Plenario2.Actions.MetaActions do
     |> Repo.insert()
   end
 
-  ##
-  # update
+  @doc """
+  Load a single row of `Meta` with any field specified by `preloads`
+  preloaded.
+
+  ## Examples
+
+    iex> get_by_pk_preload(1)
+    iex> get_by_pk_preload(1, [:data_set_constraints])
+    iex> get_by_pk_preload(1, [:data_set_constraints, :data_set_diffs])
+
+  """
+  @spec get_by_pk_preload(pk :: integer, preloads :: list) :: Meta
+  def get_by_pk_preload(pk, preloads \\ []) do
```

Comment:
Maybe you missed a merge, but there's `MetaActions.get_from_id/2` that handles this. It gets one from from a PK and handles all related preloads with a keyword list defined in `MetaQueries.handle_opts`.